apps/fluent-tester/package.json

```diff
@@ -81,32 +81,32 @@
     "appium-mac2-driver": "^1.1.4",
     "flow-bin": "^0.98.0",
     "metro-react-native-babel-preset": "^0.66.2",
-    "react": "17.0.1",
-    "react-native": "^0.64.2",
-    "react-native-macos": "^0.64.0",
-    "react-native-svg": "^12.3.0",
+    "react": "17.0.2",
+    "react-native": "^0.66.0-0",
```

Comment:
Why is the verion at the end 0-0?

Reply:
I'm not sure, I think that's what I pulled from @Saadnajmi 's branch. I can update those to match the others.

Reply:
I think this is supposed to be "Any version of RN between 0.66 and 1.0.0". This _might_ have been because the windows test app and other platform test apps were using different versions of RN. 

In hindsight, I think I'd rather keep it aligned and just do "^0.66" or whatever every other package is doing.

Reply:
Ahh I see